src/main/java/org/zaproxy/zap/extension/hud/ExtensionHUD.java

```diff
@@ -368,6 +369,7 @@ private ZapToggleButton getHudButton() {
                         hudEnabledForDesktop = hudButton.isSelected();
                         getHudParam().setEnabledForDesktop(hudEnabledForDesktop);
                         setZapCanGetFocus(!hudEnabledForDesktop);
+                        Stats.incCounter("stats.ui.toolbar.button.hud");
```

Comment:
This should be updated to match the others now.

Reply:
Fixed

Reply:
No change was pushed.

Reply:
Ugh, not sure what happened here. Will check.

Reply:
Fixed, for real 😉